scripts/ci/helpers.sh

```diff
@@ -127,9 +127,6 @@ setup_ci() {
     mkdir -p $HOME/.cedana
     echo '{"client":{"leave_running":false, "task":""}, "connection":{"cedana_url": "https://ci.cedana.ai"}}' > $HOME/.cedana/client_config.json
 
-    # Install recvtty
-    go install github.com/opencontainers/runc/contrib/cmd/recvtty@latest
```

Comment:
Why was this removed?

Reply:
https://github.com/cedana/cedana/pull/385/commits/537d1db34ed813bd44f476c3c7ec8444137abb66

Added recvtty as a debug command in cedana so we wouldn't have to deal w/ downloading it anymore